guides/source/active_record_validations.md

```diff
@@ -8,9 +8,11 @@ the database using Active Record's validations feature.
 
```

Comment:
The intro sentence seems a little bit long-winded....not sure what will be better, something like "This guide covers Active Record validation options that you can use to validate Ruby objects before saving them to the database."?